scripts/docker_run

```diff
@@ -10,6 +10,6 @@ docker run \
   --volume=$PWD:/opt/my-project \
   --workdir=/opt/my-project \
   --publish=8888:8888 \
-  --publish=30000:30000 \
+  --publish=7000-8000:7000-8000 \
```

Comment:
Nice! Could you add a comment pointing to the `node_port_min/max` flags, so that we know where these are coming from?